### PR TITLE
ISMIR 2019

### PR DESCRIPTION
**Pull request by stefan-balke**
This PR adds a template for 2019.

It includes the recent fixes with `cite.sty` (issue #14, #13), uses UTF-8 as input encoding, and T1 as font encoding (issue #15).

@f0k, @fzalkow, could you test it?

@julian-urbano please double-check all changes (also in the Word template).

### CODE REVIEW

**Comment by stefan-balke** [comment]
@f0k we do not need this any longer, correct?

```
% Optional: To use hyperref, uncomment the following.
% \usepackage[bookmarks=false,hidelinks]{hyperref}
% Mind the bookmarks=false option; bookmarks are incompatible with ismir.sty.
```

I mean, keeping it with `hidelinks` wouldn't hurt but `bookmarks=false` should no longer be needed, shouldn't it?

**Comment by fzalkow** [comment]
> @f0k, @fzalkow, could you test it?

Works fine for me, but `bookmarks=false` is still needed. (Using TeX Live 2019 on Ubuntu.)

**Comment by julian-urbano** [comment]
Thanks for doing this. Just a couple things

- Technically, it should be _**the** Netherlands_, not _Netherlands_.
- I see you added variables `\conferenceyear` and `\conferenceedition`. Should we also add something like `\conferenceplace`?

File: 2019/latex/ISMIR2019template.tex [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+\section{Equations}
+
+Equations should be placed on separate lines and numbered.
+The number should be on the right side, in parentheses, as in \eqnref{relativity}.
+
+\begin{equation}\label{relativity}
+E=mc^{2}
+\end{equation}
+
+\section{Citations}
```
Comment (ejhumphrey) [inline 2019/latex/ISMIR2019template.tex:208]: I wonder if we'd like to include scientific guidelines / reminders about e.g. citing late breaking demos, or other things discussed in  the community lately. Would this be an effective place to add this, do you think? To what extent is the paper template a submission contract, of sorts?

Comment (julian-urbano) [reply 2019/latex/ISMIR2019template.tex:208]: Yeah, I like the idea of making the templates self-contained in terms of guidelines.

Comment (stefan-balke) [reply 2019/latex/ISMIR2019template.tex:208]: I think this should be a markdown-ish thing on the conference website. Maybe we can also have on the ISMIR page. But good point though...

**Comment by stefan-balke** [comment]
@fzalkow thanks, added the comment.

@julian-urbano yes, good point, added "The Netherlands" and `\conferenceplace` macro.

I think this is ready for merge, @julian-urbano please confirm a last time.

**Comment by julian-urbano** [comment]
Looking good. Thanks!

**Pull request merged**